### PR TITLE
increase clang-tidy coverage

### PR DESCRIPTION
**Pull request by radexx99**

### CODE REVIEW

File: src/blockchain/node/filteroracle/FilterOracle.cpp [hunk 1]
```diff
@@ -659,15 +690,15 @@ auto FilterOracle::reset_tips_to(
     const cfilter::Type type,
     const block::Position& position,
     const std::optional<bool> resetHeader,
-    const std::optional<bool> resetFilter) const noexcept -> bool
+    const std::optional<bool> resetfilter) const noexcept -> bool
```
Comment (Piotrek321) [inline src/blockchain/node/filteroracle/FilterOracle.cpp]: I would leave it unchanged

File: src/blockchain/node/wallet/subchain/NotificationStateData.cpp [hunk 2]
```diff
@@ -125,7 +133,7 @@ auto NotificationStateData::handle_confirmed_matches(
         pc_display_)(" on ")(print(node_.Chain()))
         .Flush();
 
-    if (general.empty()) { return; }
+    if (0u == general.size()) { return; }
```
Comment (Piotrek321) [inline src/blockchain/node/wallet/subchain/NotificationStateData.cpp]: I would leave it unchanged

File: src/blockchain/node/wallet/subchain/NotificationStateData.cpp [hunk 3]
```diff
@@ -153,7 +161,7 @@ auto NotificationStateData::handle_mempool_matches(
 {
     const auto& [utxo, general] = matches;
 
-    if (general.empty()) { return; }
+    if (0u == general.size()) { return; }
```
Comment (Piotrek321) [inline src/blockchain/node/wallet/subchain/NotificationStateData.cpp]: I would leave it unchanged

**Review by Piotrek321 — CHANGES_REQUESTED** [review]

File: src/blockchain/node/wallet/subchain/NotificationStateData.cpp (hunk 2 above)
Comment (wiesiekpap) [reply src/blockchain/node/wallet/subchain/NotificationStateData.cpp]: A c-language programmer never dies in some people... Or is it ecmascript or something?

File: src/blockchain/node/wallet/subchain/NotificationStateData.cpp (hunk 3 above)
Comment (wiesiekpap) [reply src/blockchain/node/wallet/subchain/NotificationStateData.cpp]: If you look long enough, you'll find some dude on youtube fiercely advocating this style as a solution to all your problems :-}

**Review by wiesiekpap — APPROVED** [review]

File: src/blockchain/node/wallet/subchain/NotificationStateData.cpp (hunk 3 above)
Comment (Piotrek321) [reply src/blockchain/node/wallet/subchain/NotificationStateData.cpp]: I talked with J about that and he said this is fine for him :D (I mean chaning to .empty())

File: src/blockchain/node/filteroracle/FilterOracle.cpp (hunk 1 above)
Comment (radexx99) [reply src/blockchain/node/filteroracle/FilterOracle.cpp]: GoodPoint

File: src/blockchain/node/wallet/subchain/NotificationStateData.cpp (hunk 2 above)
Comment (radexx99) [reply src/blockchain/node/wallet/subchain/NotificationStateData.cpp]: Yes i missed

**Comment by radexx99** [comment]
New Pull Request 111 have been created to merge this work

**Pull request closed without merging**